### PR TITLE
Mixcontest 2018

### PR DESCRIPTION
**Pull request by dianguyen95**
Added Coming Soon Page

### CODE REVIEW

File: src/templates/mixcontest2018.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<script type="text/html"
+        data-route="^[Mm]ix[Cc]ontest2018"
+        data-title="Monstercat Mix Contest 2018"
+		data-process="processMixContest2018Page"
+		data-template="mixcontest2018"
+		>
+
+	{{#if loading}}
+        {{>loading-view-black}}
+    {{else}}
```
Comment (Vindexus) [inline src/templates/mixcontest2018.html]: Please fix indentations.

File: src/js/mixcontest2018.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    mixedContestScope.track = {
+      loading: true
+    }
+
+    mixedContestScope.cotwEpisodes = {
+      loading: true
+    }
+
+    // if counting down to start date
+    mixedContestScope.showStartDate = (!status.ended && !status.open) ? true : false
```
Comment (lostlevels) [inline src/js/mixcontest2018.js]: Redundant true : false, just set to ` = !status.ended && !status.open`

File: src/js/mixcontest2018.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+    // if voting is open
+    mixedContestScope.votingOpen = (status.open == true) ? true : true
```
Comment (lostlevels) [inline src/js/mixcontest2018.js]: true and true? set to `= status.open`

File: src/js/mixcontest2018.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+    // if user has voted
+    status.canVote = true // to show the poll even if account has already voted
+    mixedContestScope.hasVoted = (status.canVote == true) ? false : true
```
Comment (lostlevels) [inline src/js/mixcontest2018.js]: `!status.canVote`

**Pull request closed without merging**